ceph/rados_utils.py

```diff
@@ -221,9 +221,11 @@ def check_osd_up(self, osd_id):
         while end_time > datetime.datetime.now():
             status = self.is_up(osd_id)
             if status:
-                return 1
-            time.sleep(5)
-        return 0
+                time.sleep(5)
```

Comment:
The method returns true when the OSD is up... but this change returns true when it is down. We could fix the return code to True (or 0) when OSD is up. 

```
while end_time > datetime.datetime.now():
  if self.is_up(osd_id): return True
  time.sleep(5)

return False
```

Reply:
For interop, can we look at the code where the check is for down and negate at that location.

Reply:
The method returns true when OSD is up. ( after waiting for 120 seconds ), and returns false as soon as OSD goes down. This is expected of the method. Do we want to change the functionality?

Reply:
My bad... sorry for the misdirect. I got confused with 1 being `False`. Do we want to wait for `120` seconds to return the OSD is up?

Reply:
We wait for 120 seconds to return when OSD is down. We stay in the while loop until the said OSD is still up .